Breeding-Insight/Gigwa2
# BI-1650 - Store and fetch genotype data

# Description
**Story:** https://breedinginsight.atlassian.net/browse/BI-1650

- Removing the requirement that a DB be created from the Gigwa UI
- Change how processIDs are created for importing VCF files to not use the auth token as the processID
- Updates to the Dockerfile to support multi-architecture builds
- Updates to the docker-compose.yml file to allow for variables to be defined in an `.env` file


# Dependencies


# Testing
- Verify that Gigwa starts up
- Verify that submitting genotype data to Gigwa from the REST endpoints (not the UI) allows the uploaded data to persist longer than 1 day


# Checklist:
